prow/cmd/peribolos/main_test.go

```diff
@@ -66,33 +66,33 @@ func TestOptions(t *testing.T) {
 			name: "maximal delta",
 			args: []string{"--config-path=foo", "--maximum-removal-delta=1"},
 			expected: &options{
-				config:        "foo",
-				minAdmins:     defaultMinAdmins,
-				requireSelf:   true,
-				maximumDelta:  1,
-				logLevel:      "info",
+				config:       "foo",
```

Comment:
What are these whitespace changes? They seem unrelated

Reply:
I usually run make update-gofmt after I'm done writing code. I guess there's no mandatory job for making sure of this format

Reply:
That we don't verify that and as a result ended up with that being incorrect is an issue, but it is orthogonal to this change and should be fixed seperately

Reply:
Okay, I'll revert this change